script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_8B.txt

```diff
@@ -26,12 +26,12 @@ Arcueid glares at me with displeasure.
 That's right.
 For whatever reason, Arcueid is talking to me a lot.
 <Page9>
-If you're feeling weak then you're better off going to sleep, but
+If you're feeling weak then you're better off going to sleep but,
```

Comment:
maybe end with a dash here. The japanese structure makes it more obvious his thought process is cut off.